### PR TITLE
Master

### PR DESCRIPTION
**Pull request by adlyons**
Updating dropsonde.py to reflect changes in overall structure. Still WIP

### CODE REVIEW

File: awot/dropsondes/dropsondes.py [hunk 1]
```diff
@@ -8,651 +8,673 @@
 from skew import SkewXTick
 
 
-class DropSondes(object):
```
Comment (adlyons) [inline awot/dropsondes/dropsondes.py:11]: Got rid of the class structure and opted for functions to match latest design. Still missing a few moving parts.

File: awot/dropsondes/dropsondes.py [hunk 2]
```diff
@@ -8,651 +8,698 @@
...
+    data['Pressure'] = P
+    data['Relative Humidity'] = RH
+    data['U Component'] = U
+    data['V Component'] = V
+    data['Height'] = H
+    data['Type'] = 'dropsonde'
+
+    return data
+
+def plot_skewtlogp(data, **kwargs):
```
Comment (nguy) [inline awot/dropsondes/dropsondes.py]: Any chance you'd rename `plot_skewtlogp` to `plot_skewt_logp`?

File: awot/dropsondes/dropsondes.py [hunk 3]
```diff
@@ -8,651 +8,698 @@
...
+    data['Header'] = header
+    data['Temperature'] = T
+    data['Dewpoint'] = TD
+    data['Pressure'] = P
+    data['Relative Humidity'] = RH
+    data['U Component'] = U
+    data['V Component'] = V
+    data['Height'] = H
+    data['Type'] = 'dropsonde'
+
```
Comment (nguy) [inline awot/dropsondes/dropsondes.py]: Other components of AWOT store each variable as a dictionary. Is this possible?

File: awot/dropsondes/dropsondes.py [hunk 4]
```diff
@@ -8,651 +8,698 @@
...
+    U = u[~mask]
+    V = v[~mask]
+
+    data = dict()
+    data['Header'] = header
+    data['Temperature'] = T
+    data['Dewpoint'] = TD
+    data['Pressure'] = P
+    data['Relative Humidity'] = RH
+    data['U Component'] = U
```
Comment (nguy) [inline awot/dropsondes/dropsondes.py]: This is a personal choice, but I don't like using spaces in the name, e.g. 'U Component' to 'u_component'. I also tend to use all lower case for the keys.

File: awot/dropsondes/dropsondes.py [hunk 5]
```diff
@@ -8,651 +8,698 @@
...
-        '''
+    ax2 = fig.add_axes([.05, 0.6, 0.25, 0.3])
+
+    # create axis and invert masks and assign values
+    # for U, V, and h coordinates
+
+    U = data['U Component']
+    V = data['V Component']
+    H = data['Height']
+
```
Comment (nguy) [inline awot/dropsondes/dropsondes.py]: If we standardized the names with the other portions of AWOT (e.g. I used Uwind I think), some of these routines would become more general.

**Comment by nguy** [comment]
We'll probably want to pull some of these routines apart to fit into package architecture. For example, put the read routines in `awot.io` and then adjust the `awot.io.__initi__.py` file accordingly. Also plotting, but let's test first.

File: awot/dropsondes/__init__.py [hunk 6]
```diff
@@ -10,5 +10,10 @@
     :toctree: generated/
 
 """
+from __future__ import absolute_import
+from thermocalcs import ThermoCalcs
+from shearcalcs import ShearCalcs
+from skew import SkewXTick
+
```
Comment (nguy) [inline awot/dropsondes/__init__.py]: The imports should be relative, for example:
`from thermocalcs import ThermoCalcs` should be `from .thermocalcs import ThermoCalcs`

File: awot/dropsondes/dropsondes.py [hunk 7]
```diff
@@ -8,651 +8,676 @@
...
+    thermoCalcData['THETAE'] = THETAE
+    thermoCalcData['ESTA'] = ESAT
+
+
+return thermoCalcData
+    
+    
+
+    return
+
```
Comment (nguy) [inline awot/dropsondes/dropsondes.py]: Program bombs at L472. I assume that the first return is the valid one?

**Comment by nguy** [comment]
@adlyons I am going to merge this for now and then create another pull request with updated reader.

**Pull request merged**